src/store/slice/auth/authReducer.ts

```diff
@@ -0,0 +1,74 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { forgetPasswordService, signinService, signupService } from "./authService";
+import { AuthI } from "./types";
+
+const initialState: AuthI = {
+  loading: false,
+  user: {
+    profilePicture: "" || null,
```

Comment:
Use null as default value for any field. DO NOT USE "" or anything else. Just use null for default!!!